_layouts/register.html

```diff
@@ -253,10 +253,9 @@
       </div>
     </div>
   </div>
-
-</body>
 {% if site.google_analytics %}
 {% include google_analytics.html %}
+</body>
 {% endif %}
```

Comment:
Here and in other files: Close the endif before the closing `</body>` (it belongs to the GA snippet).

Do this in the other files, too.

The “stray end tag” issue is going to be fixed this way.